packages/ketcher-react/src/script/ui/views/modal/components/ExtendedTable/components/GenericGroups/components/ButtonGenSet/ButtonGenSet.tsx

```diff
@@ -0,0 +1,49 @@
+/****************************************************************************
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import clsx from 'clsx'
+import classes from './ButtonGenSet.module.less'
+
+const ButtonGenSet = ({
+  button,
+  disabledQueryElements,
+  onAtomSelect,
+  selected
+}) => {
+  const isDisabled = disabledQueryElements?.includes(button.label)
```

Comment:
I propose to make this component more dumb and change `disabledQueryElements` property to `disabled` and get this value from parent

Reply:
fixed